pkg/config/config.go

```diff
@@ -97,13 +100,20 @@ type TransparencyConfig struct {
 }
 
 const (
-	taskrunFormatKey  = "artifacts.taskrun.format"
-	taskrunStorageKey = "artifacts.taskrun.storage"
-	taskrunSignerKey  = "artifacts.taskrun.signer"
-
-	ociFormatKey  = "artifacts.oci.format"
-	ociStorageKey = "artifacts.oci.storage"
-	ociSignerKey  = "artifacts.oci.signer"
+	taskrunFormatKey        = "artifacts.taskrun.format"
+	taskrunStorageTektonKey = "artifacts.taskrun.storage.tekton"
+	taskrunStorageOCIKey    = "artifacts.taskrun.storage.oci"
+	taskrunStorageGCSKey    = "artifacts.taskrun.storage.gcs"
+	taskrunStorageDocDBKey  = "artifacts.taskrun.storage.docdb"
+	taskrunSignerKey        = "artifacts.taskrun.signer"
+
+	ociEnabled          = "artifact.oci.enabled"
+	ociFormatKey        = "artifacts.oci.format"
+	ociStorageTektonKey = "artifacts.oci.storage.tekton"
```

Comment:
I'm rethinking having a separate key for each backend now that I see it 🤔 

It might be difficult to keep the config organized. The advantage of a comma-separated list is that it would be backwards compatible as well.

WDYT of doing that instead?

```
artifacts.oci.storage=tekton,gcs
```

and then if the value is empty `artifacts.oci.storage=""` then we assume that we don't want to store anything at all (which would address the issue in https://github.com/tektoncd/chains/issues/279 as well)

Reply:
Yes I agree. The config does get very cluttered. Having a comma-separated would be the better way to go. Plus it would also require less unit test updates. Let me change to using that and I will re-push on this branch.

Reply:
thank you! we can keep the defaults as they are as well that way :)